### DISCUSSION TITLE
How to create FluentIcon with unsized type?

### DISCUSSION THREAD

**Discussion by ChinaChenMingQuan** [microsoft/fluentui-blazor#1990] [Q&A] [answered]
as mentioned in the title.

**Comment by oneolddev** [comment]
@ChinaChenMingQuan 

Out of curiosity, why do you need an unsized `FluentIcon`?

**Reply by ChinaChenMingQuan** [reply]
Am I to understand that this feature doesn't exist because it's not needed and will never be implemented? -- deepl translator.

**Reply by vnbaaij** [reply, marked as answer]
Indeed, that feature does not exist and we have no plans to implement.

**Reply by ChinaChenMingQuan** [reply]
I see, so I'll have to implement this feature myself. -- deepl translator.
